src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+public class Rectangle implements Figure {
+    private String color;
+    private int length;
+    private int width;
+
+    public Rectangle() {
+        this.length = length;
+        this.width = width;
+        this.color = color;
+
+    }
```

Comment:
The constructor should not have an empty body. If you intend to initialize the `Rectangle` with specific `length`, `width`, and `color`, these should be parameters of the constructor, and you should remove the default constructor if it's not used.